### PR TITLE
Fix wrong usage of == for strings

### PR DESCRIPTION
**Pull request by NolaDonato**
This bug causes the asset loader to miss textures
GearVRf-DCO-1.0-Signed-off-by: nola.donato@samsung.com Nola Donato

### CODE REVIEW

File: GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAssetLoader.java [hunk 1]
```diff
@@ -284,7 +284,7 @@ public void onAssetLoaded(GVRContext context, GVRSceneObject model, String model
         
         private void generateLoadEvent()
         {
-            if ((mScene != null) && (mModel != null) && (mErrors == ""))
+            if ((mScene != null) && (mModel != null) && (mErrors.equals("")))
```
Comment (danke-sra) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAssetLoader.java]: I think Java's recommended way is isEmpty()

File: GVRf/Framework/framework/src/main/java/org/gearvrf/jassimp2/GVRJassimpSceneObject.java [hunk 2]
```diff
@@ -206,7 +206,7 @@ private void loadTextures(GVRAssetLoader.AssetRequest assetRequest, AiMaterial m
             if(texType != AiTextureType.UNKNOWN)
             {
                 final String texFileName = material.getTextureFile(texType, 0);
-                if ((texFileName != null) && (texFileName != ""))
+                if ((texFileName != null) && !texFileName.equals(""))
```
Comment (danke-sra) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/jassimp2/GVRJassimpSceneObject.java]: same as line 287 above

Comment (liaxim) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/jassimp2/GVRJassimpSceneObject.java]: The two checks could be reduced to ``!"".equals(texFileName)``

**Comment by liaxim** [comment]
Will merge after lunch

**Comment by NolaDonato** [comment]
Not yet – I haven’t fixed the issues…

**Comment by NolaDonato** [comment]
OK to merge now I have used the clever "".equals way of comparing

**Pull request merged**